caiman_napari/_cnmf.py

```diff
@@ -76,5 +83,69 @@ def main(batch_path, uuid):
     cnmf_obj.save(uuid + '.hdf5')
 
 
+def load_output_cnmf(viewer, batch_path, uuid):
```

Comment:
I think that instead of giving the batch path, we should add another column to the back-end dataframe called `outputs`. These should be dictionaries, kinda like how the `parameters` are stored in the dataframe. The dictionaries can look like this for mcorr and cnmf:

mcor:

```
{"mcorr_output": "/path/to/mcorr_vid.mmap",
 "success": True,
 "traceback": None}
```

The `success` key can hold a boolean value. This value can be checked in the `main_offline_gui` to color the list item after an item finishes. If `success` is False, it will read the `traceback` and show it to the user.

For CNMF the dict should be similar, but instead it contains the path to the hdf5 file with the results.